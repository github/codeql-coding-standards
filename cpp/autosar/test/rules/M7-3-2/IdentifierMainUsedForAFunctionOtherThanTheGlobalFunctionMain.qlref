rules/M7-3-2/IdentifierMainUsedForAFunctionOtherThanTheGlobalFunctionMain.ql