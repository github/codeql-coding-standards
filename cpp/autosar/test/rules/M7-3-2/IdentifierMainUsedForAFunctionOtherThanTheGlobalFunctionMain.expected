| test.cpp:5:6:5:9 | main | Identifier main used for a function other than the global function main. |
| test.cpp:10:5:10:8 | main | Identifier main used for a function other than the global function main. |
| test.cpp:15:7:15:10 | main | Identifier main used for a function other than the global function main. |
| test.cpp:20:7:20:10 | main | Identifier main used for a function other than the global function main. |
